logging/testLogger.go

```diff
@@ -12,23 +16,29 @@ type testSink interface {
 	Log(...interface{})
 }
 
-// testWriter implements io.Writer and delegates to a testSink
+// testWriter implements io.Writer
```

Comment:
go-kit already has a type for exactly this purpose:
https://pkg.go.dev/github.com/go-kit/kit/log?tab=doc#NewSyncWriter

Reply:
You actually don't need to synchronize the `NopLogger`.  Just returning `NewNopLogger` is sufficient.